src/main/java/org/mule/extension/db/internal/domain/connection/oracle/OracleDbConnection.java

```diff
@@ -87,4 +106,77 @@ public OracleDbConnection(Connection jdbcConnection, List<DbType> customDataType
     }
     return tables;
   }
+
+  @Override
+  public Struct createStruct(String typeName, Object[] attributes) throws SQLException {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this method to simply inherit it. ([squid:S1185](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AS1185))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-core-extensions-mule-db-connector&pullRequest=156&issues=AWkWM61notIXnmsUtJ_a&open=AWkWM61notIXnmsUtJ_a)

[ISSUE_KEY]: AWkWM61notIXnmsUtJ_a